sbpl_lattice_planner/include/sbpl_lattice_planner/sbpl_lattice_planner.h

```diff
@@ -92,11 +102,13 @@ class SBPLLatticePlanner : public nav_core::BaseGlobalPlanner{
   std::string name_;
   costmap_2d::Costmap2DROS* costmap_ros_; /**< manages the cost map for us */
   std::vector<geometry_msgs::Point> footprint_;
+  std::vector<geometry_msgs::Point> robot_footprint_;
```

Comment:
Why is the extra copy `robot_footprint_` necessary? If I see it correctly, we could just re-use `footprint_`, right?